src/o365/spo/commands/page/page-clientsidewebpart-add.ts

```diff
@@ -341,6 +341,12 @@ class SpoPageClientSideWebPartAddCommand extends SpoCommand {
         if (webPartData.serverProcessedContent) {
           (webPart as any).serverProcessedContent = webPartData.serverProcessedContent;
         }
+        if (webPartData.dynamicDataPaths) {
+          (webPart as any).dynamicDataPaths = webPartData.dynamicDataPaths;
```

Comment:
Since you've marked `dynamicDataPaths` as public, there is no need to cast `webPart` to `any`